punchpipe/cli.py

```diff
@@ -238,15 +238,16 @@ def control_process_launcher() -> subprocess.Popen:
                 print("punchpipe abruptly shut down")
 
 
-def clean_replay(input_file: str, write: bool = True, timerange: int = 5) -> None | pd.DataFrame:
+def clean_replay(input_file: str, configuration_path, write: bool = True, timerange: int = 5) -> None | pd.DataFrame:
```

Comment:
The timerange could go in the config too. I'd like it to be called something like "window_in_days" though. Just something where the unit is included.